main.js

```diff
@@ -1 +1,36 @@
 'use strict';
+let sorting = document.getElementById('sorting');
+
+sorting.onclick = function(e) {
+  if (e.target.tagName != 'TH') return;
+  sortGrid(e.target.cellIndex, e.target.getAttribute('data-type'));
```

Comment:
will be better to destruct target from the event, do it if you need to get some object property more then once